isis/src/base/apps/isisimport/importUtils.h

```diff
@@ -48,14 +50,16 @@ namespace Isis {
                              QString processFunction,
                              PvlObject translation,
                              ProcessImport *process) {
-    vector<vector<char *>> prefixData = process->DataPrefix();
     switch (ancillaryProcessMap[processFunction.toStdString()]) {
       case 1:
         return cassiniIssCreateLinePrefixTable(cube,
-                                               prefixData.at(0),
-                                               translation);
```

Comment:
If we move the vector into the case statement we can prevent an API breaking change